mlflow-tests/tests/test_mcp_servers.py

```
@@ -0,0 +1,346 @@
+    validate_mcp_server_created,
+    validate_mcp_server_deleted,
+    validate_mcp_server_version_and_endpoint_created,
+)
+from .validations import validate_authentication_denied
+
+from mlflow_tests.enums import ResourceType, KubeVerb
+from .base import TestBase
+
+logger = logging.getLogger(__name__)
+
+_random_gen = random.Random()
```

Ruff (0.16.1)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)